CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 2.14.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.4.1166** 
```

Comment:
we will see what version is out when it is out

Reply:
Will update it to latest version right before releasing.